chapters/ch07.md

```diff
@@ -6,22 +6,22 @@
 The dev set should be large enough to detect differences between algorithms that you are trying out. For example, if classifier A has an accuracy of 90.0% and classifier B has an accuracy of 90.1%, then a dev set of 100 examples would not be able to detect this 0.1% difference. Compared to other machine learning problems I’ve seen, a 100 example dev set is small. Dev sets with sizes from 1,000 to 10,000 examples are common. With 10,000 examples, you will have a good chance of detecting an improvement of 0.1%. [2]
 
 ->
-Tập phát triển phải đủ lớn để có thể thấy được sự khác nhau giữa các thuật toán mà bạn đang thử. Ví dụ: nếu thuật toán phân loại A có độ chính xác 90,0% và thuật toán phân loại B có độ chính xác 90,1%, thì một tập có 100 mẫu sẽ không thể phát hiện sự khác biệt 0,1% này. So với các vấn đề khác trong machine learning mà tôi đã thấy, một tập phát triển với 100 mẫu là nhỏ. Các tập phát triển với kích thước từ 1.000 đến 10.000 mẫu khá phổ biến. Với 10.000 mẫu, bạn sẽ có khả năng cao để nhìn thấy mức cải thiện 0,1%. [2]
+Để nhận ra sự khác biệt giữa các thuật toán đang thử nghiệm, thì tập phát triển phải đủ lớn. Ví dụ: nếu bộ phân loại A có độ chính xác 90,0% và bộ phân loại B có độ chính xác 90,1%, thì một tập phát thiển có 100 mẫu sẽ không thể phát hiện sự khác biệt 0,1% này. So với các vấn đề khác trong machine learning mà tôi đã thấy, một tập phát triển chỉ với 100 mẫu là nhỏ. Các tập phát triển thường có từ 1,000 tới 10,000 mẫu. Với 10.000 mẫu, bạn sẽ có thể thấy sự cải thiện ở mức 0,1%. [2]
 
 For mature and important applications—for example, advertising, web search, and product recommendations—I have also seen teams that are highly motivated to eke out even a 0.01% improvement, since it has a direct impact on the company’s profits. In this case, the dev set could be much larger than 10,000, in order to detect even smaller improvements.
 
 ->
-Trong các ứng dụng quan trọng và đã khá phát triển - lấy ví dụ như quảng cáo, tìm kiếm trên web và gợi ý sản phẩm - tôi đã thấy nhiều nhóm rất muốn cải thiện thậm chí chỉ là 0,01%, vì nó có ảnh hưởng trực tiếp đến lợi nhuận của công ty. Trong trường hợp này, tập phát triển có thể lớn hơn 10.000 để phát hiện được những cải thiện còn nhỏ hơn nữa.
+Trong các ứng dụng quan trọng và đã đã đưa vào khai thác - lấy ví dụ như quảng cáo, tìm kiếm trên web và gợi ý sản phẩm - tôi đã thấy nhiều nhóm rất muốn cải thiện chất lượng thuật toán dù chỉ là 0,01%, vì nó có ảnh hưởng trực tiếp đến lợi ích của công ty. Trong trường hợp này, tập phát triển có thể lớn hơn 10.000 mẫu rất nhiều chỉ để đạt mức cải tiến còn nhỏ hơn nữa.
```

Comment:
* Sau khi mình search qua bản full *machine yearning* thì vẫn không cắt nghĩa được rõ ràng ý tác giả nói "mature application" là gì. Không rõ "ứng dụng trưởng thành" này ý là những ứng dụng đã nhiều công ty làm rồi và chất lượng đã đạt đến đỉnh rồi, nếu thêm 0,01% thì rất tốt hay muốn nói tới những ứng dụng to lớn, phức tạp. "Đã đưa vào khai thác" có lẽ mình dịch sai. Anh @tiepvupsu và @ngcthuong có thể diễn đạt từ này được không, em thấy nó còn xuất hiện mấy lần nữa trong các chương.

Reply:
- Bên ngoài bài giảng của machine learning, thì khi nói tới "mature problem" thì đều ám chỉ vấn đề mà ai cũng biết, và đã có những phải pháp rất tốt cho nó rồi. Nên mình nghĩ "mature application" cũng hiểu theo nghĩa tương tự như em nó thứ nhất. 

Ví dụ là ở bài giảng convex optimization của Stephen boy, ông có nhắc đi nhắc lại và joke về  least-squares or linear programming là mature rồi. 

Và chắc chắn không phải ám chỉ ứng dụng to lớn và phức tạp.

Reply:
@SumoBBQ anh nghĩ từ này ai cũng hiểu nhưng tiếng Việt hơi khó diễn đạt. Theo anh hiểu là những ứng dụng đã đưa vào hoạt động một thời gian đủ dài, nó không nhất thiết phức tạp nhưng có độ ổn định cao.

Có vẻ chương này chỉ còn vướng từ này nhỉ.

Reply:
Còn từ "statistical significance tests " nữa trên wiki dịch là "kiểm định giả thiết thống kê"